config/jobs/etcd/etcd-periodics.yaml

```diff
@@ -174,3 +174,168 @@ periodics:
       # fuse needs privileged mode
       securityContext:
         privileged: true
+- name: ci-etcd-performance-ratio-1-128-amd64
```

Comment:
What is our rough expected runtime for this ratio? Suggest updating pr description with a rough estimate for how long each ratio will take to run. Does not have to be perfect, just a rough estimation based on what we know so far.

Reply:
Every estimated run time (with some extra leeway) is configured in `decoration_config.timeout`. I'll update the PR description.

Reply:
Done